contributors/DEVELOPER-GUIDE.md

```diff
@@ -55,19 +55,27 @@ Once you're happy with the file contents, you'd need to reverse-engineer the cha
 
 For example, `generated.txt` would have a sibling `generated.txt.dev` file with information about the "template" and "args" files that contributed to the current result.
 
-## Back-merging main branch / Publishing to NPM (TODO: Update to main branch workflow)
+## Back-merging main branch / Publishing to NPM
 
-1. Make sure you have the latest changes from `main` branch
-2. Checkout to `cli` branch and create a new branch from it eg: `cli-backmerge`
-3. Checkout to `cli-backmerge` branch and `git merge main`
-4. If there are any conflicts, resolve them and commit the changes
-5. Add changeset by doing `yarn changeset add` follow prompt and commit changes, learn more about changeset [here](https://github.com/scaffold-eth/scaffold-eth-2/blob/cli/CONTRIBUTING.md#changeset)
-6. Push the branch and create a PR against `cli` branch
+1. Sync `upstream-main` branch with SE-2 in the GitHub UI, using "Sync fork" (you can also add the remote branch locally and do it locally)
+   <details><summary>Screenshot from GitHub UI</summary>
 
-> NOTE: The `cli-backmerge` branch should be merged with "Create a merge commit" option instead of "Squash and merge" option into `cli` branch to preserve the commit history and not needing to create an extra commit directly into `cli` to merge `main` to resolve conflicts.
+   ![gh-web-ui](https://github.com/scaffold-eth/create-eth/assets/55535804/29cd684d-bdd0-42e7-a3c1-2a6e879e1a75)
+
+   </details>
+
+2. Pull `upstream-main` branch locally
+3. Create `backmerge-upstream` branch from `main` branch
+4. Merge `upstream-main` => `backmerge-upstream` branch
+5. If there are any conflicts, resolve them and commit the changes
+   > **Common conflict:** You may need to "accept yours" in the root `package.json` file and add the `next:serve` manually to the `template\base` `package.json` file.
```

Comment:
Ohh I think, we should also mention that sometimes there will conflicts in root `yarn.lock`  file and we should "accept yours" too or reset those changes to yarn.lock

Reply:
Tweaked a bit the format, like that:

5. If there are any conflicts, resolve them and commit the changes. **Common conflicts:**
   - Sometimes there will be conflicts in the root `yarn.lock` file and we should "accept yours", or reset the changes made to `yarn.lock`.
   - You may also need to "accept yours" in the root `package.json` file and manually add `next:serve` to the `template\base` `package.json` file.